ui/tests/acceptance/settings/auth/configure/section-test.js

```diff
@@ -37,7 +37,7 @@ module('Acceptance | settings/auth/configure/section', function (hooks) {
     const section = 'options';
     await enablePage.enable(type, path);
     await page.visit({ path, section });
-    await page.fillInTextarea('description', 'This is AppRole!');
+    await fillIn('[data-test-input="description"]', 'This is Approle!');
```

Comment:
mostly an FYI - there is a general selectors file that has these common selectors 

https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.js#L46